src/View/UIViewTrait.php

```diff
@@ -33,5 +33,8 @@ public function initializeUI(array $options = [])
         $this->loadHelper('Form', ['className' => 'BootstrapUI.Form']);
         $this->loadHelper('Flash', ['className' => 'BootstrapUI.Flash']);
         $this->loadHelper('Paginator', ['className' => 'BootstrapUI.Paginator']);
+        if (class_exists('\Cake\View\Helper\BreadcrumbsHelper')) {
```

Comment:
We'd have to drop php 5.4 support if we didn't include this, as the BreadcrumbsHelper was only introduced in CakePHP 3.3.6.